<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title>BC2_2023.knit</title>







<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->
<link href='https://fonts.googleapis.com/css?family=Special Elite' rel='stylesheet'>
<link href='https://fonts.googleapis.com/css?family=Rasa' rel='stylesheet'>

<!--
Here are the required style attributes for css to make this poster work :)
-->
<style>
@page {
size: 33.1in 46.8in;
margin: 0;
padding: 0;
}
body {
margin: 0px;
padding: 0px;
width: 33.1in;
height: 46.8in;
text-align: justify;
font-size: 45px;
line-height: 1.05;
}
/* RMarkdown Class Styles */
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
margin: auto;
margin-top: 1em;
margin-bottom: 1em;
/* align-items: center; */
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
code {
font-size: 25pt;
font-family: monospace;
background-color: #00808024;
color: #0b4545;
padding: 1.2mm;
line-height: 1;
border-radius: 2mm;
}
caption {
margin-bottom: 10px;
font-size: 20pt;
font-style: italic;
}

tbody tr:nth-child(odd) {
    background-color: #00808020;
}
.table>thead>tr>th, .table>tbody>tr>th, .table>tfoot>tr>th, .table>thead>tr>td, .table>tbody>tr>td, .table>tfoot>tr>td{
  border-spacing: 0;
  font-size: 40%;
  border-style: none;
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 1em;
  padding-left: 1em;
  line-height: 1em;
}
table {
  margin: auto;
}
th {
  padding-left: 5mm;
  padding-right: 5mm;
}
.caption {
font-size: 20pt;
font-style: italic;
padding-top: 0;
}
.references {
font-size: 15px;
line-height: 90%;
}
/* Create three unequal columns that floats next to each other */
.column {
float: left;
padding: 0px;
}
.outer {
width: 33.1in;
height: calc(46.8in *  (1 - 0.115 - 9.5e-2 - 0.01) );
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
column-gap: 0;
padding-left: 0cm;
padding-right: 0cm;
/* -webkit-column-rule-width: 50%;
-moz-column-rule-width: 50%;
column-rule-width: 50%; */
-webkit-column-rule-style: none;
-moz-column-rule-style: none;
column-rule-style: none;
-webkit-column-rule-color: black;
-moz-column-rule-color: black;
column-rule-color: black;
background-color: #ffffff;
font-family: Rasa;
margin-top: calc(46.8in *  0.115 );
padding-top: 1em;
padding-bottom: 1em;
}
span.citation {
  color: #008080;
  font-weight: bold;
}
a {
text-decoration: none;
color: #008080;
}
#title {
font-size: 125pt;
text-align: left;
margin: 0;
line-height: 98%;
border-bottom: 0;
font-weight: normal;
background: 0;
}
#author {
color: #0b4545;
margin: 0;
line-height: 85%;
font-size: 1.17em;
}
#affiliation {
padding-top: 0.1em;
color: ;
font-style: italic;
font-size: 25px;
margin: 0;
}
sup {
color: #cc0000;
}
.affiliation sup {
font-size: 20px;
}
.author {
text-align: left;
}
.author sup {
font-size: 30px;
}
.author_extra {
color: #008080;
margin: 0;
line-height: 85%;
font-size: 35px;
text-align: left;
}
.outer h1, h2, h3, h4, h5, h6 {
text-align: center;
margin: 0;
font-weight: bold;
}
.section h1 {
  text-align:center;
  padding-bottom:5px;
  background:
    linear-gradient(
      to left,
      #ffffff 1%,
      #ffffff 20%,
      #0b454575 33%,
      #0b4545 50%,
      #0b454575 66%,
      #ffffff 80%,
      #ffffff 99%
    )
    left
    bottom
    #ffffff
    no-repeat;
  background-size:100% 5px ;
  margin-top: 0.5em;
  margin-bottom: 0.5em;
}
.outer h2 {
text-align: center;
}
.outer p, .level2 {
color: #000000;
}
.outer ol {
padding-left: 8%;
padding-right: 8%;
text-align: left;
}
.main {
width: 33.1in;
height: calc(46.8in * 0.115);
position: absolute;
background-color: #0b4545;
color: #ffffff90;
font-family: Special Elite;
background-image: linear-gradient(#0b4545 50%,#008080);
}
.main strong {
color: #ffffff;
}
.main strong > sup {
color: #ffffff;
}
.main sup {
color: #ffffff90;
}
#main-img-left {
width: 10%;
left: 0.5in;
bottom: 0.2in;
position: absolute;
}
#main-img-center {
width: 10%;
left: calc(33.1in * 0.45);
bottom: 0.5in;
position: absolute;
}
#main-img-right {
width: 10%;
right: 0.5in;
bottom: 0.2in;
position: absolute;
}
.main p {
font-size: 150px;
font-family: Special Elite;
text-align: center;
margin: 0;
position: absolute;
top: 50%;
-ms-transform: translateY(-50%);
transform: translateY(-50%);
margin-left: 1em;
}
.fab {
color: #00000030;
font-size: 25px;
}
.twitter, i {
color: #00000030;
font-size: 35px;
text-decoration: none;
}
a.email {
text-decoration: none;
color: #00000030;
font-size: 35px;
}
.envelope {
color: #00000030;
font-size: 5px;
text-decoration: none;
}
.poster_wrap {
width: 33.1in;
height: 46.8in;
padding: 0cm;
}
.main_bottom {
width: 33.1in;
height: calc(46.8in * 9.5e-2);
margin-top: calc(46.8in * (1 - 9.5e-2));
position: absolute;
background-color: #0b4545;
background-image: linear-gradient(#008080 10%, #0b4545);
}
.section {
  padding-left: 10mm;
  padding-right: 10mm;
}
span > #tab:mytable {
  font-weight: bold;
}
.orcid img {
  width: 3%;
}
.emphasis {
  background-color: #008080;
  color: #ffffff;
  border: solid #0b2045 3mm;
  margin: 1em;
  padding-left: 0;
  padding-right: 0;
}
.emphasis h1 {
  font-weight: bold;
  background: none;
  background-color: #0b2045;
  padding-bottom: 5mm;
  padding-top: 1mm;
  margin-top: -1mm;
  margin-right: -1mm;
  margin-left: -1mm;
}
.emphasis blockquote {
  border: 0;
}
.emphasis ol {
  padding: 0;
  padding-left: 8%;
  font-size: 100%;
  font-weight: bold;
}
.emphasis p {
  color: #ffffff;
}
</style>
</head>
<body>


<div class="poster_wrap">

<div class="column outer">
<div class="section">

<h3 id="author" class="author">

<strong>Léonard Hérault</strong><sup> 1,2, <a class="orcid" href="https://orcid.org/0000-0001-6499-2991"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup><br>

<a class='envelope'><i class="fas fa-envelope"></i></a> <a href="mailto:leonard.herault@unil.ch" class="email">leonard.herault@unil.ch</a> <br>
   </h3>

<h5 id="author_extra", class="author_extra">
 Aurélie Gabriel<sup>1,2</sup>
 Mariia Bilous<sup>1,2</sup>
 David Gfeller<sup>1,2</sup>
</h5>


<p id="affiliation" class="affiliation">
<sup>1</sup> Department of Oncology, Ludwig Institute for Cancer research, University of Lausanne<br> <sup>2</sup> Swiss Institute of Bioinformatics
</p>
</div>

<style>
#main-img-left {
 width: 13%;
}
#main-img-center {
 width: 10%;
}
#main-img-right {
 width: 13%;
}
</style>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<ul>
<li>Single-cell multiomics: measurement of different modalities (e.g., ATAC, RNA, proteins) in the same cell (Fig.<a href="#fig:benchmark">3</a>A).</li>
<li>Precise analysis of cell-type specific transcriptional regulation (Fig.<a href="#fig:benchmark">3</a>B).</li>
</ul>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:intro"></span>
<img src="images/intro.jpg" alt="Single-cell multiomics (A) can be used to study cell-type specific transcriptional regulation (B)." width="100%" />
<p class="caption">
Figure 1: Single-cell multiomics (A) can be used to study cell-type specific transcriptional regulation (B).
</p>
</div>
<ul>
<li><strong>Limitations:</strong> Large size, high sparsity of the data.</li>
<li><strong>Solution:</strong> Merging highly similar cells in metacells, proposed for scRNA-seq <span class="citation">(<a href="#ref-baran_metacell_2019" role="doc-biblioref">Baran et al. 2019</a>)</span>.</li>
<li><strong>Aim:</strong> Extension of <code>SuperCell</code> <span class="citation">(<a href="#ref-bilous_metacells_2022-1" role="doc-biblioref">Bilous et al. 2022</a>)</span>, to single-cell multiomics. (Fig.<a href="#fig:workflow">2</a>B).</li>
</ul>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:workflow"></span>
<img src="images/workflow.jpg" alt="SuperCell workflow to identify multiomics metacells at a graining level γ. The multimodal knn graph is computed using the WNN method from Seurat [@hao_integrated_2021]." width="100%" />
<p class="caption">
Figure 2: SuperCell workflow to identify multiomics metacells at a graining level γ. The multimodal knn graph is computed using the WNN method from Seurat <span class="citation">(<a href="#ref-hao_integrated_2021" role="doc-biblioref">Hao et al. 2021</a>)</span>.
</p>
</div>
</div>
<div id="benchmark" class="section level1">
<h1>Benchmark</h1>
<p>Multiomodal version of <code>SuperCell</code> versus unimodal tools (Fig.<a href="#fig:benchmark">3</a>):</p>
<ul>
<li>Purer metacells</li>
<li>Compact and separated metacells in both modalities</li>
<li>Faster</li>
</ul>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:benchmark"></span>
<img src="BC2_2023_files/figure-html/benchmark-1.png" alt="**A** Benchmark metrics. **B** Benchmark results of metacell tools on a 10x multiome (RNA + ATAC) dataset of PBMCs, graining level γ=75. Tools: new version of SuperCell, SEACells [@persad_seacells_2023], MetaCell2 [@ben-kiki_metacell-2_2022]." width="100%" />
<p class="caption">
Figure 3: <strong>A</strong> Benchmark metrics. <strong>B</strong> Benchmark results of metacell tools on a 10x multiome (RNA + ATAC) dataset of PBMCs, graining level γ=75. Tools: new version of SuperCell, SEACells <span class="citation">(<a href="#ref-persad_seacells_2023" role="doc-biblioref">Persad et al. 2023</a>)</span>, MetaCell2 <span class="citation">(<a href="#ref-ben-kiki_metacell-2_2022" role="doc-biblioref">Ben-Kiki et al. 2022</a>)</span>.
</p>
</div>
</div>
<div id="supercell-analyses" class="section level1">
<h1>SuperCell Analyses</h1>
<div id="x-multiome-dataset-of-pbmcs" class="section level3">
<h3>10X multiome dataset of PBMCs</h3>
<p><code>SuperCell</code> identifies robust metacells in the PBMC multiomic space (Fig.<a href="#fig:pbmcmulti">4</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:pbmcmulti"></span>
<img src="BC2_2023_files/figure-html/pbmcmulti-1.png" alt="Identified metacells in the multiomic space of PBMCs, graining level γ = 75." width="100%" />
<p class="caption">
Figure 4: Identified metacells in the multiomic space of PBMCs, graining level γ = 75.
</p>
</div>
<p>Gene accessibility and expression appear more correlated at the metacell level (Fig.<a href="#fig:crMultiome">5</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:crMultiome"></span>
<img src="BC2_2023_files/figure-html/crMultiome-1.png" alt="**A**. Gene accessibility - gene expression correlation for TCF7 and SPI1. Left: Single-cell level, right: metacells γ = 75. Same color legend as in Fig.\@ref(fig:pbmcmulti). **B**. Same correlations for the 2000 highly variable genes (on RNA) with increasing γ." width="100%" />
<p class="caption">
Figure 5: <strong>A</strong>. Gene accessibility - gene expression correlation for TCF7 and SPI1. Left: Single-cell level, right: metacells γ = 75. Same color legend as in Fig.<a href="#fig:pbmcmulti">4</a>. <strong>B</strong>. Same correlations for the 2000 highly variable genes (on RNA) with increasing γ.
</p>
</div>
<p>Correlation between transcription factor (TF) expression and corresponding motif accessibility also becomes clearer using metacells (Fig.<a href="#fig:crMotifRna">6</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:crMotifRna"></span>
<img src="BC2_2023_files/figure-html/crMotifRna-1.png" alt="**A.** TF expression (RNA) - motif accessibility (ATAC) correlation for TCF7 and SPI1. Left: single-cell, right: metacells, γ = 75. Same color legend as in  Fig.\@ref(fig:pbmcmulti). **B.** Same correlations for 200 TFs (with cor &gt; 0.01 in single-cells) with increasing γ." width="100%" />
<p class="caption">
Figure 6: <strong>A.</strong> TF expression (RNA) - motif accessibility (ATAC) correlation for TCF7 and SPI1. Left: single-cell, right: metacells, γ = 75. Same color legend as in Fig.<a href="#fig:pbmcmulti">4</a>. <strong>B.</strong> Same correlations for 200 TFs (with cor &gt; 0.01 in single-cells) with increasing γ.
</p>
</div>
<p><br />
</p>
</div>
<div id="cite-seq-atlas-of-160000-pbmcs" class="section level3">
<h3>CITE-seq atlas of 160,000 PBMCs</h3>
<ul>
<li>Identification of metacells by sample (γ = 20).</li>
<li>Correction of the batch effect at the metacell level (Fig.<a href="#fig:citeAtlas">7</a>A&amp;B).</li>
<li>This workflow runs on a standard laptop (Fig.<a href="#fig:citeAtlas">7</a>C).</li>
</ul>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:citeAtlas"></span>
<img src="BC2_2023_files/figure-html/citeAtlas-1.png" alt="**A.** UMAP visualizations of 8,000 multiomic metacells, metacell purities  with respect to original annotation from [@hao_integrated_2021]. **B.** Metacell workflow. **C.** Computational resources used by single-cell and metacell workflows." width="100%" />
<p class="caption">
Figure 7: <strong>A.</strong> UMAP visualizations of 8,000 multiomic metacells, metacell purities with respect to original annotation from <span class="citation">(<a href="#ref-hao_integrated_2021" role="doc-biblioref">Hao et al. 2021</a>)</span>. <strong>B.</strong> Metacell workflow. <strong>C.</strong> Computational resources used by single-cell and metacell workflows.
</p>
</div>
<p>RNA-protein correlation in the CITE-seq atlas is increased with metacells (Fig.<a href="#fig:globalCrCiteSeq">8</a>).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:globalCrCiteSeq"></span>
<img src="BC2_2023_files/figure-html/globalCrCiteSeq-1.png" alt="**A.** RNA-protein correlation for CD55 and CD19 in the 8 reference samples. Left: Single-cell level, right: metacells γ = 20 Same color legend as in Fig.\@ref(fig:citeAtlas). **B.** RNA-Protein correlation for 213 gene-protein pairs with increasing γ." width="100%" />
<p class="caption">
Figure 8: <strong>A.</strong> RNA-protein correlation for CD55 and CD19 in the 8 reference samples. Left: Single-cell level, right: metacells γ = 20 Same color legend as in Fig.<a href="#fig:citeAtlas">7</a>. <strong>B.</strong> RNA-Protein correlation for 213 gene-protein pairs with increasing γ.
</p>
</div>
</div>
</div>
<div id="conclusion" class="section level1">
<h1>Conclusion</h1>
<p><img src="images/conclusion.jpg" width="100%" style="display: block; margin: auto;" /></p>
<div id="perspectives" class="section level2">
<h2>Perspectives</h2>
<ul>
<li><strong>Downstream analyses</strong> of sc-multiomics:
<ul>
<li>gene regulatory network inference</li>
<li>multiomic velocity</li>
</ul></li>
<li><strong>Metaspots</strong> for spatial-omics</li>
</ul>
<div id="references" class="section level3 unnumbered">
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-baran_metacell_2019" class="csl-entry">
Baran, Yael, Akhiad Bercovich, Arnau Sebe-Pedros, Yaniv Lubling, Amir Giladi, Elad Chomsky, Zohar Meir, Michael Hoichman, Aviezer Lifshitz, and Amos Tanay. 2019. <span>“<span>MetaCell</span>: Analysis of Single-Cell <span>RNA</span>-Seq Data Using <span>K</span>-Nn Graph Partitions.”</span> <em>Genome Biology</em> 20 (1): 206. <a href="https://doi.org/10.1186/s13059-019-1812-2">https://doi.org/10.1186/s13059-019-1812-2</a>.
</div>
<div id="ref-ben-kiki_metacell-2_2022" class="csl-entry">
Ben-Kiki, Oren, Akhiad Bercovich, Aviezer Lifshitz, and Amos Tanay. 2022. <span>“Metacell-2: A Divide-and-Conquer Metacell Algorithm for Scalable <span class="nocase">scRNA</span>-Seq Analysis.”</span> <em>Genome Biology</em> 23 (1): 100. <a href="https://doi.org/10.1186/s13059-022-02667-1">https://doi.org/10.1186/s13059-022-02667-1</a>.
</div>
<div id="ref-bilous_metacells_2022-1" class="csl-entry">
Bilous, Mariia, Loc Tran, Chiara Cianciaruso, Aurélie Gabriel, Hugo Michel, Santiago J. Carmona, Mikael J. Pittet, and David Gfeller. 2022. <span>“Metacells Untangle Large and Complex Single-Cell Transcriptome Networks.”</span> <em>BMC Bioinformatics</em> 23 (1): 336. <a href="https://doi.org/10.1186/s12859-022-04861-1">https://doi.org/10.1186/s12859-022-04861-1</a>.
</div>
<div id="ref-hao_integrated_2021" class="csl-entry">
Hao, Yuhan, Stephanie Hao, Erica Andersen-Nissen, William M. Mauck, Shiwei Zheng, Andrew Butler, Maddie J. Lee, et al. 2021. <span>“Integrated Analysis of Multimodal Single-Cell Data.”</span> <em>Cell</em> 184 (13): 3573–3587.e29. <a href="https://doi.org/10.1016/j.cell.2021.04.048">https://doi.org/10.1016/j.cell.2021.04.048</a>.
</div>
<div id="ref-persad_seacells_2023" class="csl-entry">
Persad, Sitara, Zi-Ning Choo, Christine Dien, Noor Sohail, Ignas Masilionis, Ronan Chaligné, Tal Nawy, et al. 2023. <span>“<span>SEACells</span> Infers Transcriptional and Epigenomic Cellular States from Single-Cell Genomics Data.”</span> <em>Nature Biotechnology</em>, March, 1–12. <a href="https://doi.org/10.1038/s41587-023-01716-9">https://doi.org/10.1038/s41587-023-01716-9</a>.
</div>
</div>
</div>
</div>
</div>

</div>
<div class="main">
<p><strong>Metacells</strong> facilitate the analysis of single-cell <strong>multiomics</strong> data</p>
</div>
<div class="main_bottom">
<img id="main-img-left" src=images/Poster/logoLeftFF.jpg>
<img id="main-img-center" src=images/Poster/qr-code.png>
<img id="main-img-right" src=images/Poster/logoRightFF.jpg>
</div>
</div>



</body>
</html>
